include/zephyr/bluetooth/bluetooth.h

```diff
@@ -2125,6 +2125,16 @@ struct bt_le_scan_cb {
 					   BT_GAP_SCAN_FAST_INTERVAL, \
 					   BT_GAP_SCAN_FAST_WINDOW)
 
+/**
+ * @brief Helper macro to enable active scanning to discover new devices with window == interval.
+ *
+ * Continuous scanning should be used to maximize the chances of receiving advertising packets.
+ */
+#define BT_LE_SCAN_ACTIVE_CONTINUOUS BT_LE_SCAN_PARAM(BT_LE_SCAN_TYPE_ACTIVE, \
+						      BT_LE_SCAN_OPT_FILTER_DUPLICATE, \
+						      BT_GAP_SCAN_FAST_WINDOW, \
```

Comment:
Define:
```
#define BT_GAP_SCAN_FAST_INTERVAL_MIN               0x0030  /* 30 ms    */
```


```suggestion
						      BT_GAP_SCAN_FAST_INTERVAL_MIN, \
```

Reply:
We do not have MIN and MAX defines for scan windows and intervals, do still believe I should them?
We have them for advertising because there the controller can choose some value between min and max.

Reply:
BT Spec has recommended min and max range for the scan interval, to be consistent we should relate to the specification's recommended values of min and max value:
![image](https://github.com/zephyrproject-rtos/zephyr/assets/6350656/87f265ac-8840-4c36-9421-84da53921d01)

Reply:
It would still make sense to decide on if we want to use 30 or 60 ms as the scan interval.

Reply:
> This sounds out of scope for this PR :)

Supplying `BT_GAP_SCAN_FAST_WINDOW`  name to scan interval parameter in a public interface is incorrect.

> It would still make sense to decide on if we want to use 30 or 60 ms as the scan interval.

- For windowed scanning, we keep the current `BT_GAP_SCAN_FAST_INTERVAL` value of 30 ms, no public interface change.
- For continuous scanning, we define new `BT_GAP_SCAN_FAST_INTERVAL_MIN` of 30 ms

Reply:
I suggest doing as @cvinayak says and adding `BUILD_ASSERT(BT_GAP_SCAN_FAST_WINDOW == BT_GAP_SCAN_FAST_INTERVAL_MIN, "Continuous scanning is requested by setting window and interval equal.")` next to the definition of `BT_LE_SCAN_ACTIVE_CONTINUOUS`.

Please also leave a comment on whether it matters what the value is or if they just have to be equal.

Reply:
@Thalley , we could do that. But I assume that in most cases the user doesn't really care about the scan window. Therefore I have a slight preference for @cvinayak 's and @alwa-nordic 's proposal